plugin/metrics/vars/vars.go

```diff
@@ -59,6 +59,12 @@ var (
 		Name:      "response_rcode_count_total",
 		Help:      "Counter of response status codes.",
 	}, []string{"server", "zone", "rcode"})
+
+	Panic = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: plugin.Namespace,
+		Name:      "panic_count_total",
+		Help:      "A metrics that counts the number of panics.",
```

Comment:
Are there any labels we can add in for the panic to help identify the cause? Not sure if we could identify a plugin/pkg at the point of panic to add in as a label.

Reply:
not sure, the error plugin had a whole bunch of stuff to show this, but I'm sorta hoping the stack trace lands in your logging frame work...